### PR TITLE
Add filter elements for styling

### PR DESCRIPTION
**Pull request by WillMoggridge**
Couple of steps to this PR, for continuing styling. Proposing to filters branch, not master.

First, I abstracted the header section from the main app file. I also moved the SVG out because it clutters the page!

Then I added some basic elements. The search field is in `codelabs-index.html`, as it will likely want to be styled at the side of the text.
The other filter elements can be found in `elements/tutorials-filters.html`.

The select boxes are using `paper-dropdown-menu` but this can be changed to a less opinionated option to make styling easier.
Some example info for `iron-dropdown` if you want to switch: http://stackoverflow.com/questions/31729942/how-to-use-iron-dropdown-in-polymer-1-0

### CODE REVIEW

File: src/elements/tutorials-filters.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-item/paper-item.html">
+<link rel="import" href="../../bower_components/paper-listbox/paper-listbox.html">
+<link rel="import" href="../../bower_components/paper-tabs/paper-tab.html">
+<link rel="import" href="../../bower_components/paper-tabs/paper-tabs.html">
+
+<dom-module id="tutorials-filters">
+  <template>
+    <style>
+    </style>
```
Comment (anthonydillon) [inline src/elements/tutorials-filters.html]: Do you need an empty style section?

**Comment by WillMoggridge** [comment]
Yeah, this is for a feature branch for continued work. The logic is not attached to the form in this PR. Graham will be filling style block in with following work.

That QA issue is annoying though...

**Comment by WillMoggridge** [comment]
Oh, I wonder if the error is from needing to `bower install`.

I will be much more careful about including notes on that in the future.

**Comment by WillMoggridge** [comment]
OK, I realised the problem was z-index. I think I have sorted that now.

**Comment by anthonydillon** [comment]
Approving but note the styling is broken.

**Pull request merged**